### PR TITLE
Hooks added to the project

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
Updated Calculator.JSX to use hooks.
Reviewed main App either.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -7,120 +7,124 @@ import {
...
-    super(props);
-    this.state = {
-      results: null,
-      next: null,
-      operation: null,
-      valueToShow: 0,
-    };
-    this.clickHandler = this.clickHandler.bind(this);
-  }
+function Calculator() {
```
Comment (Gambit142) [inline src/components/Calculator.jsx:10]: - [x] (Optional) Consider converting this function to `Es6` function. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @fernando-silvabr66 👋, 

Good job so far!

### Highlights
- Use of functional Components ✔️ 
- Use of React Hooks ✔️ 
- All linters checks passed ✔️ 

However, there are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Requires changes ♻️ 

- Kindly customize your `README.md` file and use a personalized one. The one you have is the README.md that came with CRA. You can use that of Microverse and edit it with some commands from the _**CRA**_ `README.md` file if you wish But it is good practice and a general requirement in Microverse to personalize your `README.md` file. 👍 

### Optional Changes
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider the suggestion I made under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Calculator.jsx (hunk 1 above)
Comment (fernando-silvabr66) [reply src/components/Calculator.jsx:10]: Will also work on that suggestion in the continuity of the project... thanks for the suggestion.

**Review by Meltrust — APPROVED** [review]
Hi @fernando-silvabr66,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the customized readme file, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fernando-silvabr66** [comment]
Really thanks for that review... 🙌

**Pull request merged**